### PR TITLE
Add lacp stp selecting root bridge test

### PR DESCRIPTION
**Pull request by StepanVovkPLV**
- Adds 2 test
 - Adds addition option `mstid `to MstPctl.set()

### CODE REVIEW

**Review by yuriilisovskyi — APPROVED** [review]

File: DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/functional/stp/test_stp_select_root.py [hunk 1]
```diff
@@ -0,0 +1,323 @@
...
+    assert out[0][dent]['parsed_output'][0]['root-port'] == '', f'Bridge { bridge_names[0]} is not a root bridge'
+    # Other bridges do not consider themselves as root bridge
+    for bridge in bridge_names:
+        out = await Mstpctl.show(input_data=[{dent: [
+                {'parameter': 'bridge',
+                 'bridge': bridge,
+                 'options': '-f json'}]}], parse_output=True)
+        assert out[0][dent]['rc'] == 0, 'Failed to get bridge detail'
+        assert out[0][dent]['parsed_output'][0][
+                   'root-port'] != '', f'Bridge {bridge_names[1]} or {bridge_names[2]} is a root bridge'
```
Comment (SerhiyBoikoPLV) [inline DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/functional/stp/test_stp_select_root.py:122]:
```suggestion
        assert out[0][dent]['parsed_output'][0]['root-port'] != '', \
            f'Bridge {bridge_names[1]} or {bridge_names[2]} is a root bridge'
```

File: DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/functional/stp/test_stp_select_root.py [hunk 2]
```diff
@@ -0,0 +1,323 @@
...
+    # 10. Wait for topology to re-build.
+    await asyncio.sleep(wait_time)
+
+    # 11.Verify bridge_3 has become the new root bridge.
+    out = await Mstpctl.show(input_data=[{dent: [
+        {'parameter': 'bridge',
+         'bridge': bridge_names[2],
+         'options': '-f json'}]}], parse_output=True)
+    assert out[0][dent]['rc'] == 0, 'Failed to get bridge detail'
+    assert out[0][dent]['parsed_output'][0]['root-port'] == '', f'Bridge { bridge_names[2]} is a root bridge'
```
Comment (SerhiyBoikoPLV) [inline DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/functional/stp/test_stp_select_root.py:160]:
```suggestion
    assert out[0][dent]['parsed_output'][0]['root-port'] == '', f'Bridge {bridge_names[2]} is a root bridge'
```

File: DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/functional/stp/test_stp_select_root.py [hunk 3]
```diff
@@ -0,0 +1,323 @@
...
+    # a priority value less than the default set
+    out = await Mstpctl.set(input_data=[{dent: [
+        {'parameter': 'treeportprio',
+         'bridge': bridge_names[0],
+         'port': list(loopback_ports.values())[1],
+         'mstid': 0,
+         'priority': 7}
+        ]}], parse_output=True)
+    assert out[0][dent]['rc'] == 0, 'Failed setting bridge priority'
+    # assert out[0][dent]['rc'] == 0, 'Failed to change priority of the bridge'  # RC = '-1', but command succeeds
```
Comment (SerhiyBoikoPLV) [inline DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/functional/stp/test_stp_select_root.py:305]: uncomment or open issue

**Review by SerhiyBoikoPLV — CHANGES_REQUESTED** [review]

**Comment by StepanVovkPLV** [comment]
https://github.com/PLVision/dent-testing/pull/164 all tests here

**Pull request closed without merging**